paasta_tools/instance/kubernetes.py

```diff
@@ -584,24 +642,46 @@ def get_pod_status(pod: V1Pod, backends: Optional[Set[str]],) -> Dict[str, Any]:
         "message": message,
         "scheduled": scheduled,
         "ready": ready,
-        "containers": get_pod_containers(pod),
+        "mesh_ready": mesh_ready,
+        "containers": await get_pod_containers(pod, client, num_tail_lines),
```

Comment:
It might be worth trying to do the `get_pod_containers` and `get_pod_event_messages` at the same time and awaiting both at the same time

Reply:
I didn't find a quick & easy way to convert this to make both of these calls in parallel from `get_pod_status`, so I think I will work on that as a separate improvement.

Reply:
no worries -- probably not worth working too hard on it anyway